src/sentry/static/sentry/app/views/onboarding/createProject.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import {browserHistory, Link} from 'react-router';
+import SentryTypes from '../../proptypes';
+
+import ApiMixin from '../../mixins/apiMixin';
+import ProjectActions from '../../actions/projectActions';
+import {getPlatformName} from './utils';
+
+import OnboardingProject from '../onboarding/project';
+import {t} from '../../locale';
+
+import Raven from 'raven-js';
+
+const CreateProject = React.createClass({
+  propTypes: {
+    getDocsUrl: React.PropTypes.func
+  },
+
+  contextTypes: {
+    organization: SentryTypes.Organization,
```

Comment:
Ah by `OrganizationState`, I meant this mixin https://github.com/getsentry/sentry/blob/e2e221fa9cde5b00035ef8bc9386bde71bd721eb/src/sentry/static/sentry/app/mixins/organizationState.jsx